trillian-logserver.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: trillian-logserver
```

Comment:
this isn't necessarily a blocker but I wondered if we should have one top level package `trillian` and then multiple subpackages, one for each `./cmd/...` in [here](https://github.com/google/trillian/tree/master/cmd)

Reply:
Yeah, I'd recommend we do this, I just refactored `rekor` to do exactly this.

Reply:
https://github.com/wolfi-dev/os/pull/6021/files

Reply:
I wouldn't use a range for 2, I'd do what rekor does :)

Reply:
thanks, i think i fixed it!